src/components/app/Home.js

```diff
@@ -78,6 +102,44 @@ class UploadButton extends React.PureComponent<UploadButtonProps> {
           />
           Select a profile to open
         </label>
+        <label
+          id="LoadFromUrlButton"
+          className="homeSectionButton"
+          onClick={this._loadFromUrlPressed}
+        >
+          Load profile from an URL
+        </label>
```

Comment:
Now it would be very nice if this button could have the "pressed" style if `isLoadFromUrlPressed` is true, and set `isLoadFromUrlPressed` back to `false` if the user presses the button again.

Reply:
yes it should be a button but when i make it button its text's look differ from load from file button, the space between line become smaller